hazenlib/ACRObject.py

```diff
@@ -227,16 +226,19 @@ def measure_orthogonal_lengths(self, mask):
         """
         dims = mask.shape
         dx, dy = self.pixel_spacing
+        [centre, radius] = self.find_phantom_center(self.images[slice_index])
```

Comment:
I think these can be unpacked directly in one line, like
`[(vertical, horizontal), radius] = self.find_phantom_centre(...)`